Clone of TNW web page

Zappat0n opened the pull request:
This is a pull request for merging the TheNextWeb clone to the main branch.

Zappat0n committed:
Added files to initialize project

Zappat0n committed:
Added sections and top navigation bar HTML

o-rashad committed:
adding index and stlye files structure for mobile version

o-rashad committed:
Styling latest-news Mob version

o-rashad committed:
Adding latest-funding

o-rashad committed:
Updating Structure

Zappat0n committed:
Added styles to funding and topics

o-rashad committed:
Adding and Styling Topics and Latest Deals sections

Zappat0n committed:
Added footer and started with Ipad version

o-rashad committed:
Styleing tab version

o-rashad committed:
Styling the Nav medium screen

o-rashad committed:
Styling Main section and navs in medium media

Zappat0n committed:
Added styles for latest news and latest funding

o-rashad committed:
Updating the Medium and Large screens

Zappat0n committed:
Added styles to latest fundings, topics and latest deals

o-rashad committed:
Fixing Styles issues in Medium, Large and Larger screens

Zappat0n committed:
Fixed hint errors and added content

o-rashad committed:
Fixing styles Latest funding section

Zappat0n committed:
Fixed linter errors

Zappat0n committed:
Fix margin error

Zappat0n committed:
Add files via upload

Zappat0n committed:
Update README.md

Zappat0n committed:
Update README.md

Zappat0n committed:
Update README.md

uryelah commented on styles.css:
```diff
@@ -0,0 +1,673 @@
+
+/* <!-- MAIN SECTION --> */
+
+#main-section {
```
- [x] Avoid using id selectors to style elements in CSS. [Check this article with reasons for why that's a bac practice](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) and change the selectors to class selectors(you can always [combine selectors to be have more specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) if that was the reason for using ids).

od-c0d3r replied:
Thank you @uryelah 

Very Professional, descriptive, and clean code review ⭐⭐⭐📝

We are working on it and would push it ASAP...

@od-c0d3r
@Zappat0n

uryelah reviewed (changes requested):
Hi **Rashad Rashad & Angel Barros Pazos**, I'm Sarah your reviewer for this PR 👩‍🚀 

You guys did a great job building a very responsive clone of the Next Web page and nailing your semantic HTML. There are some minor issues to fix to do with the CSS code style and how some sections of the page look on larger screens but you are really close to the finish line so fear not!

Read the review comments below carefully and take some time to compare your clone with the [original page](https://thenextweb.com/), using your browser responsiveness tools, and then get to coding!

If you think you need a refresher about how to use responsiveness tools I'd check [this Firefox, the best browser for developers, turorial](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode).

Rock on! 🎸 👨‍🎤 

# Required Changes 🏗️ 🐛 💡

-----

 **Kindly fix the pointed out items and comments below before requesting the next code review**


- [x] Avoid using id as style selectors in CSS, prefer class selectors, check the line comment below this message.

- [x] Improve the footer at medium to large screens in such a way that it utilizes the same shades of grey that the original one does, has similar type fonts to the original under the icons, and that the bottom area text is all in the same line, not broken in three lines.
If you are not sure how to get the same shade of grey the original site uses you can just use the [developer tools inspector of your browser](https://developers.google.com/web/tools/chrome-devtools/css) and get the colors from the element styles or use a [color picker like the one Firefox has](https://developer.mozilla.org/en-US/docs/Tools/Eyedropper)
- [x] Make sure that on large screens the sections below the main articles at wide screens all take the same width and are aligned horizontally like on the original page:

**Compare the origin site on larger screens and your clone in the screenshots below:**

 <table style="width:100%">
  <tr>
    <th>Original Page</th>
    <th>Clone</th>
  </tr>
  <tr>
    <td valign="top"><img src=https://user-images.githubusercontent.com/16193212/97618531-e4254480-19fd-11eb-8f68-d67d89a35eec.png /></td>
    <td><img src=https://user-images.githubusercontent.com/16193212/97618526-e25b8100-19fd-11eb-8bd2-b5bebd3ee969.png /></td>
  </tr>
<tr>
<td>
  <img src=https://user-images.githubusercontent.com/16193212/97618668-133bb600-19fe-11eb-9417-7787fa2f56ed.png />
</td>
<td>
  <img src=https://user-images.githubusercontent.com/16193212/97618672-146ce300-19fe-11eb-813b-f26126325b9c.png />
</td>
</tr>
</table> 

---

- [x] [OPTIONAL] Let's add some box-shadow under the fixed top navbar to make it look closer to the original one and make it 'pop' in the eye by giving it some depth.

- [x] [OPTIONAL] Why don't you add an icon to the tab of your clone to make the page more professional? Check [this StackOverflow thread about it if you are not sure how to achieve it.](https://stackoverflow.com/questions/4888377/how-to-add-a-browser-tab-icon-favicon-for-a-website)

- [x] [OPTIONAL] A lot of the links in the original page have [hover effects](https://css-tricks.com/almanac/selectors/h/hover/) that change the text color or add underlines, why don't you play a bit with hover effects and transitions? That really improves the page UX(user experience) and makes it feel more 'real'.

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

o-rashad commented:
Thank you @uryelah 

Very professional, descriptive, and oriented Code review ⭐️⭐️⭐️📝

We are currently working on fixing it and would be pushing it ASAP...

@Zappat0n , @od-c0d3r

o-rashad committed:
UPDATE after 1st Code Review.

Based on the last code review instructions
we finished the below :
1- IDs selectors in CSS file has been changed to classes selectors.
2- Footer section has been improved.
3- Hover with transtion effects has been implemented.
4- Latest News section overall margins has been improved.
5- Shadow effect has been added to the top Navbar.

o-rashad committed:
Update README.md

o-rashad committed:
Fixing Linters Errors

o-rashad committed:
Merge branch 'new-branch' of https://github.com/Zappat0n/TNW-clone into new-branch

o-rashad committed:
Update .gitignore

o-rashad commented:
### **UPDATE after 1st Code Review.**

**Based on the last code review instructions
we finished the below :**

1- IDs selectors in CSS file has been changed to classes selectors.
2- Footer section has been improved.
3- Hover with transtion effects has been implemented.
4- Latest News section overall margins has been improved.
5- Shadow effect has been added to the top Navbar.

Thanks to @uryelah

o-rashad committed:
Fixing margins

o-rashad committed:
Updating .gitignore file for local machine linters

uche-inyama reviewed (approved):
## Approved

### Hi team :wave: 

Good job, go on with your next task.

Just an observation, you missed adding the logo for screen size 1024px. As you scroll down through the page the logo should appear.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

o-rashad commented:
Good Catch 🕵. @uche-inyama 

We gonna try to fix it push it and merge the pull request. 📝

Thank you for the CR 🙋‍♂️ @uche-inyama , @uryelah ...

Zappat0n committed:
Fixed styles in footer and linter errors
